web/src/list_widget.ts

```diff
@@ -428,7 +435,7 @@ export function create<Key = unknown, Item = Key>(
                 opts.$parent_container.off("click.list_widget_sort", "[data-sort]");
             }
 
-            if (opts.filter && opts.filter.$element) {
+            if (opts.filter?.$element) {
                 opts.filter.$element.off("input.list_widget_filter");
             }
```

Comment:
Should the if statement here be removed in favor of the next line using the `?`?